webClient/src/app/shared/dialog/save-to/save-to.component.scss

```diff
@@ -11,6 +11,38 @@
 mat-dialog-actions {
     justify-content: flex-end;
 }
+
+.mat-form-field {
+margin-left: 30px;
+}
+
+.align-label {
+  margin-left: 36px;
+}
+
+.align-encoding {
+  margin-left: 34px;
+}
+
+.cancel-button {
+  border: none;
+  min-width: 88px;
+  line-height: 36px;
+  padding: 0 16px;
+  background: white;

```

Comment:
Can you change this to "background: transparent;" as to make this scale better in case the UI changes color?

Reply:
Sure, will update the pr once

Reply:
Done